src/tools/rest-backend/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+find_package (CppCMS)
+find_package (Boost 1.50 COMPONENTS system filesystem thread REQUIRED)
```

Comment:
Why is boost required? Is booster from cppcms not enough? Why do you need filesystem access?